packages/react/src/runtimes/edge/createEdgeRuntimeAPI.ts

```
@@ -135,43 +158,46 @@
+              ];
+              onFinish({
+                messages: resultingMessages,
                 metadata: {
-                  unstable_data: lastChunk.metadata?.unstable_data ?? [],
-                  unstable_annotations:
-                    lastChunk.metadata?.unstable_annotations ?? [],
-                  steps: lastChunk.metadata?.steps ?? [],
-                  custom: lastChunk.metadata?.custom ?? {},
+                  // TODO
+                  // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
+                  steps: lastChunk.metadata?.steps!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)